pkg/allocation/converters/converter.go

```diff
@@ -64,7 +64,7 @@ func ConvertAllocationRequestToGSA(in *pb.AllocationRequest) *allocationv1.GameS
 	}
 
 	if ls := convertLabelSelectorToInternalLabelSelector(in.GetRequiredGameServerSelector()); ls != nil {
-		gsa.Spec.Required = *ls
+		gsa.Spec.Required = allocationv1.GameServerSelector{LabelSelector: *ls}
```

Comment:
This code will get updates once again once the proto changes, but this allows current functionality to continue working as expected.